cpp/src/strings/convert/convert_floats.cu

```diff
@@ -316,7 +319,7 @@ struct ftos_converter {
       return 3;
     }
     bool bneg = false;
-    if (value < 0.0) {
+    if (signbit(value)) {  // handles -0.0 too
```

Comment:
Possibly out of scope for this PR, but should we differentiate between quiet and signaling NaNs too? It sounds like `signbit()` returns true for signaling NaNs, I think flipping the sign here will make it quiet?

Reply:
There is an `std::isnan(value)` check before this line. It will output only `NaN` regardless of `bneg`.

I've not seen any library print `sNaN` before nor `-NaN` so I'm inclined to wait until that becomes a requirement. Also, I could not get Pandas to recognize `"sNaN"` and using `"-NaN"` and it just converts it to `NaN`.